shard-operator-parent/shard-operator-v1/src/main/java/com/redhat/service/smartevents/shard/operator/v1/resources/BridgeIngressStatus.java

```diff
@@ -24,6 +24,7 @@ public class BridgeIngressStatus extends CustomResourceStatus {
 
     private static final HashSet<Condition> INGRESS_CONDITIONS = new HashSet<>() {
         {
+            add(new Condition(ConditionTypeConstants.AUGMENTING, ConditionStatus.True));
```

Comment:
Setting it to `UNKNOWN` to begin with _may_ be better (as you set it to `True` in the reconcile loop).

At *this point* it is not being augmented and you could introduce a (difficult to debug) issue.

WDYT?

Reply:
done